src/js/components/header/mobile.js

```diff
@@ -9,20 +9,26 @@ function toggleClass (element, className) {
   }
 }
 
-function mobileInit () {
-  var navElements = select('.usa-menu-btn, .usa-overlay, .usa-nav-close');
+function handleNavElements (e) {
+
   var toggleElements = select('.usa-overlay, .usa-nav');
   var navCloseElement = select('.usa-nav-close')[ 0 ];
```

Comment:
That's a good point @shawnbot. But, I can't be sure what the `e.target` is going to be here right? I don't think there will be multiple navigations on the page. I think other parts of the codebase also rely on the idea of including elements globally rather than contextually.